src/client/sky.cpp

```diff
@@ -95,7 +94,6 @@ Sky::Sky(s32 id, RenderingEngine *rendering_engine, ITextureSource *tsrc, IShade
 
 	for (int i = 5; i < 11; i++) {
 		m_materials[i] = baseMaterial();
-		m_materials[i].Lighting = true;
```

Comment:
Removing this luckily doesn't seem to have any effect (used for texture skybox rendering)